sdk/nodejs/tests/runtime/langhost/run.spec.ts

```diff
@@ -1136,6 +1136,24 @@ describe("rpc", () => {
                 return { urn: makeUrn(t, name), id: undefined, props: undefined };
             },
         },
+        // Create a resource with a large string to test grpcMaxMessageSize increase.
+        "large_resource": {
+            program: path.join(base, "065.large_resource"),
+            expectResourceCount: 1,
+            registerResource: (ctx: any, dryrun: boolean, t: string, name: string, res: any) => {
+                const longString = "a".repeat(1024 * 1024 * 5);
+                assert.strictEqual(t, "test:index:MyLargeStringResource");
+                assert.strictEqual(name, "testResource1");
+                assert.deepEqual(res, { "largeStringProp": longString });
+                return {
+                    urn: makeUrn(t, name),
+                    id: name,
+                    props: {
+                        "largeStringProp": "a".repeat(1024 * 1024 * 5),
+                    },
+                };
+            },
+        },
```

Comment:
Do these tests actually go through the RPC layer?  I don't recall off the top of my head - but I feel like they do not?  cc @pgavlin 

One place we could add tests to be sure they hit the RPC is in `/test/integration`.  Those will run the full "normal" CLI/Engine/LangHost setup.

Reply:
BTW - same likely applies to other languages.

Reply:
@lukehoban yes they do. I made sure by checking that they fail on master with the same resource exhausted error. I can definitely add integration tests too but I am certain that these tests go through the langhost <-> engine RPC layer.